tableize.go

```diff
@@ -13,6 +15,7 @@ type Input struct {
 	// Optional
 	HintSize      int
 	Substitutions map[string]string
+	StringifyArr  bool
```

Comment:
Weird to shorten "Array" to "Arr", but not "Stringify". Can we change this to `StringifyArrays bool`?